runtime/src/main/cpp/Natives.h

```diff
@@ -136,6 +136,10 @@ OBJ_GETTER0(Kotlin_konan_internal_undefined);
 void Kotlin_konan_internal_GC_suspend(KRef);
 void Kotlin_konan_internal_GC_resume(KRef);
 
+// Random operations.
+KInt Kotlin_random();
```

Comment:
I'd say that you'd better avoid using system's RNG and have consistent cross-platform implementation instead.